### DISCUSSION TITLE
Training performance of TExt detection model FCENET degrading on increasing epochs on CTW1500 style custom dataset

### DISCUSSION THREAD

**Discussion by manusid123** [open-mmlab/mmocr#1638] [Q&A]
We are trying to train text detection model **FCENET** on custom dataset which is of the style of CTW1500 dataset with curved text on objects but the object is focused one. 

We tried to train **FCENET** for **500** epochs ona  custom dataset of **4k** images with some augmentations but we found that pre-trained model is giving better performance than trained models. 

Also hmean precision and recall are also decreasing with increasing epochs

Could anyone please suggest whether we are doing something wrong on training side ?

**Comment by qiyueliuhuo** [comment]
I also seem to encounter such a problem. The more the training effect is, the worse the effect. I am trained on another data set.
